apps/web/features/customization/components/widget-preview.tsx

```
@@ -28,96 +27,82 @@ export const WidgetPreview = ({ data }: WidgetPreviewProps) => {
+				)}
 			</div>
 			{!isOpen && (
-				<motion.button
-					initial={{ scale: 0 }}
-					animate={{ scale: 1 }}
-					whileHover={{ scale: 1.1 }}
+				<button
 					onClick={() => setIsOpen(true)}
-					className="absolute bottom-0 right-0 flex h-14 w-14 items-center justify-center rounded-full bg-primary text-primary-foreground shadow-lg"
+					className="absolute bottom-0 right-0 flex h-14 w-14 items-center justify-center rounded-full bg-primary text-primary-foreground shadow-lg hover:scale-105 transition-transform"
 				>
```

Biome (2.4.6)

[error] Provide an explicit type prop for the button element.

(lint/a11y/useButtonType)